.github/workflows/librarian.yaml

```diff
@@ -27,7 +27,11 @@ jobs:
     - name: Display Go version
       run: go version
     - name: Run tests
-      run: go test ./...
+      run: go test -coverprofile=coverage.out ./...
```

Comment:
how does this test work? when I run it locally it seems to just print the coverage percentage, but never fails

Reply:
> how does this test work? when I run it locally it seems to just print the coverage percentage, but never fails

I've added the codecov config but it didn't fail the workflow when the coverage is below 90%.

According to this [step](https://docs.codecov.com/docs/github-2-getting-a-codecov-account-and-uploading-coverage), we may need to merge this pull request first.